release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1559,6 +1559,36 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
 
+* Previously, when you imported a devfile, the Openshift web console ignored the `build guidance` placeholder container because the new deployment had a second container. The web console could not start the container because it could not fetch the placeholder image. As a result, the user container missed the additional configurations. Now, in the web console, the `build guidance` container is dropped from the new deployment, and the container adds the environment variable, ports, and limit configurations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952214[BZ#1952214])
+
+* Previously, the *Topology* view showed an error or a blank page. This happened because the OpenShift Samples Operator loads page templates to the OpenShift namespace, and was not installed. The current release fixes this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[BZ#1949810])
+
+* Previously, routes tied to the *Developer* perspective were not rendered. This produced 404 error messages when you refresh the Developer perspective. The current release fixes this issue by loading all the inactive routes. Now, the routes switch to the correct perspective before loading. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929769[BZ#1929769])
+
+* Previously, the colors for the Low and Medium severity issues of the Image Manifest Vulnerabilities (IMVs) did not match the color representation shown in the (link:https://quay.io/[Quay.io]) interface. As a result, when the user changed the severity order of vulnerabilities to High, the IMVs ordered the issues incorrectly. This created confusion when reviewing the IMVs. The current release fixes this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942716[BZ#1942716])
+
+* Previously, if it encountered a conflict while updating a resource quota, the API server sometimes failed to create a resource and returned a 409 status code. In that case, the client had to retry the API request. With this update, when the API server encounters this type of conflict, it retries creating the resource up to three times. This is often sufficient to create the resource. Otherwise, the API server displays an error in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1920699[BZ#1920699])
+
+* Previously, *Pipelines* you created in the git import flow for private repos failed to run.
+This happened because *Pipeline* ServiceAccount did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline ServiceAccount's annotations, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[BZ#1970470])
```

Comment:
```suggestion
This happened because the *Pipeline* `ServiceAccount` object did not use secrets created by git import flow for private git repositories. With this update, you can add a secret name to the pipeline `ServiceAccount` object's annotations, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970470[BZ#1970470])
```

Reply:
Are Pipeline and Pipelines bold because they are UI elements?

Reply:
repos/repositories
*Pipelines*/pipelines
git/Git in all the instances, please
add a secret name to the pipeline `ServiceAccount` object's annotations/add a secret name to the annotations of the pipeline `ServiceAccount` object

Reply:
- did not use secrets created by git import flow ---> did not use secrets created by the git import flow
- pipeline `ServiceAccount` object's annotations ---> annotations of the Pipeline `serviceAccount` object (Or) annotations of the Pipeline ServiceAccount object

Please check and use the correct way on how to represent this ^.

Reply:
Done